recipes/volume_extend.rb

```diff
@@ -0,0 +1,63 @@
+#
+# Cookbook Name:: gluster
+# Recipe:: server_setup
+#
+# Copyright 2015, Andrew Repton
+# Copyright 2015, Biola University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+node['gluster']['server']['volumes'].each do |volume_name, volume_values|
+  # 1. Get the current size of the logical volume
+  # 2. Compare to the size set for the gluster volume
+  # 3. If different, run a resize action against that volume
+  # ToDO: change hardcoded VG name gluster into an attribute
+  require 'lvm'
```

Comment:
I suspect this is an issue with your wrapper or your environment, the cookbook by itself is working fine in all the tests I'm doing (including the lvm cookbook in server.rb should install of the required gems). Does it work without your wrapper? Do you have another version of the LVM cookbook uploaded in your environment?